addons/point_of_sale/static/src/js/gui.js

```diff
@@ -357,27 +357,32 @@ var Gui = core.Class.extend({
 
     /* ---- Gui: FILE I/O ---- */
 
-    // This will make the browser download 'contents' as a 
-    // file named 'name'
-    // if 'contents' is not a string, it is converted into
-    // a JSON representation of the contents. 
-
-    download_file: function(contents, name) {
-        var URL = window.URL || window.webkitURL;
+    debug_email: function(debug_type,date_string,data) {
+        var self = this;
+        var email_to = this.pos.get_cashier().email || "";
+        var subject_line = "[POS][DEBUG] Support "+debug_type;
         
-        if (typeof contents !== 'string') {
-            contents = JSON.stringify(contents,null,2);
-        }
-
-        var blob = new Blob([contents]);
-
-        var evt  = document.createEvent("HTMLEvents");
-            evt.initEvent("click");
+        var body = "<p>Hello,</p>"+
+                        "<p>Please find the export of debug type ["+debug_type+"]</p>"+
```

Comment:
so the type is either 'Export' or 'Traceback'? Maybe something a bit more userfriendly like "please find the context of your {export,error}"